include/functions.php

```diff
@@ -0,0 +1,48 @@
+<?php
+    function getPost(){
```

Comment:
This code all looks good, but I'd suggest tweaking the naming a bit. Pay attention to pluralization. I'd call this function `getPosts` and I'd rename `SeePost` to `getPost`. That way you have two functions with similar names, but one is pluralized to indicate that you're getting all of the posts, and the other is singular to indicate that you're just getting one.